charts/timed/values.yaml

```diff
@@ -31,11 +27,19 @@ backend:
     # requests:
     #  cpu: 100m
     #  memory: 128Mi
+  settings:
+    # django settings for backend
+    emailUrl: smtp://localhost:25
+    emailFrom: webmaster@chart-example.local
+    serverEmail: webmaster@chart-example.local
+
+
```

Comment:
Remove 2 of these three newlines please.

Reply:
done and added newlines between root objects